README.md

```diff
@@ -12,6 +12,10 @@ This is a Monorepo managed using [Turborepo](https://turbo.build/) and contains
 
 1. [React-sdk](apps/react-sdk): AsyncAPI React SDK is a set of components/functions to use React as render engine for the generator. This is the library that undestand components from Generator's templates that are configured to use `react` render engine. 
 
+1. [Generator-helper](packages/helpers): A utility library that provides helper functions and utilities to simplify template development for baked-in templates. It reduces boilerplate and speeds up template creation.
+
+1. [Generator-component](packages/components): A library of reusable components that can be shared across different baked-in templates, helping to avoid duplication and accelerate template development.
```

Comment:
```suggestion
1. [Generator-helpers](packages/helpers): A utility library that provides helper functions and utilities to simplify template development. It reduces boilerplate and speeds up template creation.

1. [Generator-components](packages/components): A library of reusable components that can be shared across different templates, helping to avoid duplication and accelerate template development.
```

they are published to npm separately cause they actually can be also used by any template, even though developed outside generator repo